lib/capistrano/tasks/docker/compose.rake

```diff
@@ -81,6 +81,7 @@ namespace :docker do
   def compose_build_command
     cmd = ["build"]
     cmd << fetch(:docker_compose_build_services) unless fetch(:docker_compose_build_services).nil?
+    cmd.unshift("-f #{fetch(:docker_compose_path)}")
```

Comment:
The -f option, according to docker-compose, must be added just after docker-compose command,

For instance: `docker-compose -f {docker_compose_path} -p {docker_compose_project_name} build {docker_compose_build_services}`

So this unshift should be added at the very last lines